drivers/hud_apr/app/models/hud_apr/generators/shared/fy2024/question_twenty_three.rb

```diff
@@ -59,26 +59,26 @@ def q23e_destination_type_by_race_and_ethnicity
         # homeless
         sheet.update_cell_members(
           cell: [letter, 2],
-          members: group_scope.where(a_t[:destination].in([101, 116, 118])),
+          members: group_scope.where(a_t[:destination].in(100..199)),
         )
         # institutional
         sheet.update_cell_members(
           cell: [letter, 3],
-          members: group_scope.where(a_t[:destination].in([215, 206, 207, 225, 204, 205])),
+          members: group_scope.where(a_t[:destination].in(200..299)),
         )
         # temporary
         sheet.update_cell_members(
           cell: [letter, 4],
-          members: group_scope.where(a_t[:destination].in([302, 329, 314, 332, 312, 313, 327])),
+          members: group_scope.where(a_t[:destination].in(300..399)),
         )
         # permanent
         sheet.update_cell_members(
           cell: [letter, 5],
-          members: group_scope.where(a_t[:destination].in([422, 423, 426, 410, 435, 421, 411])),
+          members: group_scope.where(a_t[:destination].in(400..499)),
         )
         sheet.update_cell_members(
           cell: [letter, 6],
-          members: group_scope.where(a_t[:destination].in([8, 9, 17, 24, 30, 99]).or(a_t[:destination].eq(nil))),
+          members: group_scope.where(a_t[:destination].in(HudUtility2024.other_destinations).or(a_t[:destination].eq(nil))),
```

Comment:
Is `HudUtility2024.other_destinations` equivalent to `1..99`?  Maybe we should follow the pattern above?  (That said, if it's a random list of ids, then maybe we should delegate to the HUD utility.

Reply:
I've updated this to 1..99 in the most recent push